src/wallet/wallet.h

```diff
@@ -1158,6 +1165,11 @@ class CWallet final : public WalletStorage, private interfaces::Chain::Notificat
         m_last_block_processed_height = block_height;
         m_last_block_processed = block_hash;
     };
+    /** Get if chain is initial block download */
+    bool IsIBD() EXCLUSIVE_LOCKS_REQUIRED(cs_wallet)
```

Comment:
naming: perhaps `isInitialBlockDownload` if you want to follow the current convention